visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -1248,10 +1248,17 @@ void MotionPlanningDisplay::load(const rviz::Config& config)
   if (frame_)
   {
     QString host;
-    if (config.mapGetString("MoveIt_Warehouse_Host", &host))
+    ros::NodeHandle nh;
+    std::string host_param;
+    if (nh.getParam("warehouse_host", host_param))
+    {
+      host = QString::fromStdString(host_param);
+      frame_->ui_->database_host->setText(host);
+    }
+    else if (config.mapGetString("MoveIt_Warehouse_Host", &host))
       frame_->ui_->database_host->setText(host);
     int port;
-    if (config.mapGetInt("MoveIt_Warehouse_Port", &port))
+    if (nh.getParam("warehouse_port", port) || config.mapGetInt("MoveIt_Warehouse_Port", &port))
```

Comment:
The requests consists of multiple orthogonal patches.
I do believe most of them can be merged into indigo without any problem.
Let's see what it looks like once it's cleaned. :-)

Concerning this particular patch, I do not believe it is actually a reasonable
behavior to ignore the item in the rviz config if the rosparam is available,
so this part should probably be changed anyway. It's probably easy to make it
backward-compatible. E.g. one could fill in the values only when they are not
specified/empty in the config.